static/js/app.js

```diff
@@ -0,0 +1,286 @@
+$(document).ready(function() {
+    updateCart();
+    $('.cart-section').hide();
+    $('.header-list-mob').hide();
+
+    //Add Event Listeners
+    $('#cartIcon').click(onCartItem);
+    $('#mobLogo').click(showList);
+    $('#headerNavList').click(hideList);
+    $('#checkoutBtn').click(proceedToCheckout);
+});
+
+var isMobileOrTablet = false;
+(function isMobileTablet() {
+    if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)) {
+        isMobileOrTablet = true
+    }
+})();
+
+function onCartItem() {
+    if (isMobileOrTablet) {
+        return window.location.pathname = 'cart'
+    }
+    $('#myModal').show();
+    renderCartItems();
+}
+
+function showList() {
+    $('.header-list-mob').show();
+}
+
+function hideList() {
+    $('.header-list-mob').hide();
+}
+
+function renderCartData(data) {
+    let rawTemplate = document.getElementById("cart-template").innerHTML;
+    let compiledTemplate = Handlebars.compile(rawTemplate);
+    let generatedHTML = compiledTemplate(data);
+    document.getElementById("cart-modal-container").innerHTML = generatedHTML;
+}
+
+function renderCartItems() {
+    fetch('http://localhost:3200/getProducts')
+        .then(response => response.json())
```

Comment:
create a separate service file for ajax calls. Import urls from a config or constant file. Don't hard code urls

Reply:
Addressed.